lmfit/minimizer.py

```diff
@@ -33,19 +34,15 @@
 from .parameter import Parameter, Parameters
 
 #  scipy version notes:
-#  currently scipy 0.14 is required.
+#  currently scipy 0.15 is required.
 #  feature           scipy version added
 #    minimize              0.11
 #    OptimizeResult        0.13
 #    diff_evolution        0.15
 #    least_squares         0.17
 
-# differential_evolution is only present in scipy >= 0.15
-try:
-    from scipy.optimize import differential_evolution as scipy_diffev
-except ImportError:
-    from ._differentialevolution import differential_evolution as scipy_diffev
 
+from scipy.optimize import differential_evolution as scipy_diffev
```

Comment:
why not `from scipy.optimize import differential_evolution`? Is it necessary to bind it to the `scipy_diffev` name?